index.html

```diff
@@ -1,236 +1,221 @@
+                <h4>Source Code</h4><br>
+                <p>Are you a Java programmer?</p>
+                <p>Want to contribute to the most ground-breaking project the Minecraft community has ever seen</p>
+                <p>or just love testing stuff to destruction</p>
+                <p>then writing long and detailed bug reports about it?</p>
+                <p>Come get involved on <a href="https://github.com/codecrafter47/rainbow/">GitHub</a></p>
+ 
             </div>
         </div>
-    </article>
-</div>
-
-<!-- Credits -->
-<div class="wrapper style4">
-    <article id="contact" class="container 75%">
-        <ul id="copyright">
-            <li>Page created by Nicholas Mahoney & Isaiah Patton</li>
-            <li>Logo by BlackScorpion</li>
-            <li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
-        </ul>
-    </article>
-</div>
-
-<!-- Scripts -->
-<script src="assets/js/jquery.min.js"></script>
-<script src="assets/js/jquery.scrolly.min.js"></script>
-<script src="assets/js/skel.min.js"></script>
-<script src="assets/js/skel-viewport.min.js"></script>
-<script src="assets/js/util.js"></script>
-<!--[if lte IE 8]>
-<script src="assets/js/ie/respond.min.js"></script><![endif]-->
-<script src="assets/js/main.js"></script>
+        <div class="w3-third">
+            <div class="w3-card-2 w3-padding-top" style="min-height:360px">
+                <span class="fa fa-wrench"></span>
+                <h4>Plugin Maker</h4><br>
+                <p>Are you a Java programmer?</p>
+                <p>What to make an plugin (aka addon/mod) for Rainbow?</p>
+                <p>Have a look at our PluginAPI located in the Rainbow.zip</p>
+                <p>Or look at our 
+                    <a href="http://ci.codecrafter47.dyndns.eu/job/Rainbow/org.projectrainbow$PluginReference/javadoc/" class="w3-btn">Javadocs</a>&nbsp;!
+                </p>
+            </div>
+        </div>
+        <div class="w3-third">
+            <div class="w3-card-2 w3-padding-top" style="min-height:360px">
+                <span class="fa fa-comments"></span>
+                <h4>Forums</h4><br>
+                <p>Looking for something more casual?</p>
+                <p>Somewhere you can hang out with fellow Rainbow users?</p>
+                <p>Critique new plugin ideas and help people out with Rainbow?</p>
+                <p>Look no further than the <a href="http://www.project-rainbow.org/site/" class="w3-btn">Rainbow Forums.</a></p>
+            </div>
+        </div>
+    </div>
+    
+    <div class="rainbow-padding" id="footer"></div>
 
+    <!-- Footer -->
+    <footer class="w3-container w3-teal">
+        <p class="w3-center">
+            Page created by <a href="https://isaiahpatton.github.io/">Isaiah Patton</a>.
+            Using <a href="http://www.w3schools.com/w3css/default.asp" target="_blank">w3.css</a><br>
+            Rainbow Logo by BlackScorpion<br>
+            <a href="http://bugaos.weebly.com">
```

Comment:
I don't think this is an appropriate place to advertise unrelated third-party projects.